code/controllers/verbs.dm

```diff
@@ -76,6 +75,30 @@
 		if("Vote")
 			debug_variables(vote)
 			feedback_add_details("admin_verb","DVote")
+		if("Diseases")

```

Comment:
I don't think having these be a part of the controller debug verb's a bad thing. I didn't even know you could click these in the DI tab. Having another access point certainly doesn't hurt.